mkdocs_click/_docs.py

```diff
@@ -100,19 +106,59 @@ def _make_usage(ctx: click.Context) -> Iterator[str]:
     yield ""
 
 
-def _make_options(ctx: click.Context) -> Iterator[str]:
+def _make_options(ctx: click.Context, style: str = "plain") -> Iterator[str]:
     """Create the Markdown lines describing the options for the command."""
-    formatter = ctx.make_formatter()
-    click.Command.format_options(ctx.command, ctx, formatter)
-    # First line is redundant "Options"
-    # Last line is `--help`
-    option_lines = formatter.getvalue().splitlines()[1:-1]
-    if not option_lines:
-        return
-
-    yield "Options:"
-    yield ""
-    yield "```"
-    yield from option_lines
-    yield "```"
-    yield ""
+
+    def backquote(opts: Iterable[str]) -> List[str]:
+        return [f"`{opt}`" for opt in opts]
+
+    def format_possible_value(opt: click.Option) -> str:
+        param_type = opt.type
```

Comment:
(Nit) Maybe let's define a dedicated `display_name` variable that we can reuse throughout the possible formattings here?

```suggestion
        param_type = opt.type
        display_name = param_type.name.upper()
```

Reply:
No objection, but I'm considering defining `display_name` as `param_type.name`. I uppercased the name in accordance with [the original click](https://github.com/pallets/click/blob/4aabded1badf94b1de8bb939f11cbebcdd94a22e/src/click/core.py#L1926). However, I suppose there isn't much significance in doing so because the dedicated Type columns already makes the type annotation distinctive enough unlike the plain format (and uppercases looks a bit too busy IMHO). What do you think about dropping the call of `upper()` here?

Reply:
Yup, don't really mind dropping `.upper()`. That would give `choice` and `datetime`,  instead of `CHOICE` and `DATETIME`, right?

Reply:
Yes, that's right.